File: Source/Gamemodes/Standard/Standard.cs
public static class Standard
{
    public const float ROUND_INTERMISSION = 10.0f;

    public static void Initialize()
    {
        ShadowKitty.Initialize();
        Difficulty.Initialize();
        Windwalk.Initialize();
        ProtectionOfAncients.Initialize();
        SpawnChampions.Initialize();
        RollerSkates.Initialize();
        EasterEggManager.LoadEasterEggs();
        Relic.RegisterRelicEnabler();
    }

    public static void SetupProgressive()
    {
        Gamemode.NumberOfRounds = 3;
    }

    public static void SetupStandard()
    {
        Gamemode.NumberOfRounds = 5;
    }
}

File: src/Gamemodes/Standard/Standard.ts
import { SpawnChampions } from 'src/Game/Entities/Champions/SpawnChampions'
import { ShadowKitty } from 'src/Game/Entities/ShadowKitty'
import { Relic } from 'src/Game/Items/Relics/Relic'
import { RollerSkates } from 'src/Game/Items/RollerSkates'
import { ProtectionOfAncients } from 'src/Game/ProtectionOfAncients'
import { RoundManager } from 'src/Game/Rounds/RoundManager'
import { Difficulty } from 'src/Init/Difficulty/Difficulty'
import { EasterEggManager } from 'src/Rewards/EasterEggs/EasterEggManager'
import { AntiblockWand } from 'src/SpecialEffects/AntiblockWand'
import { Windwalk } from 'src/SpecialEffects/Windwalk'
import { PROD } from 'src/env'

export class Standard {
    private static ROUND_INTERMISSION: number = 10.0

    public static Initialize() {
        RoundManager.ROUND_INTERMISSION = !PROD ? 0.0 : this.ROUND_INTERMISSION
        ShadowKitty.Initialize()
        Difficulty.Initialize()
        Windwalk.Initialize()
        ProtectionOfAncients.Initialize()
        SpawnChampions.Initialize()
        RollerSkates.Initialize()
        EasterEggManager.LoadEasterEggs()
        AntiblockWand.Initialize()
        Relic.RegisterRelicEnabler()
    }
}
